playbooks/ansible/roles/sit.gpfs/templates/smb_share.conf.j2

```diff
@@ -1,6 +1,6 @@
 [{{ name }}-{{ config.be.name }}-{{ config.be.variant }}]
 comment = Volume '{{ name }}' from {{ config.be.name }}({{ config.be.variant }})
-vfs objects = acl_xattr
+vfs objects = acl_xattr {%- if config.be.variant == 'vfs' %} gpfs{% endif +%}
 path = {{ path }}
```

Comment:
So the `path` value(_/mnt/share_) remains the same and is valid in this case?

Reply:
Yes. The VFS module in this case is just a helper to provide more features or improve efficiency, but most of the I/O still goes through the kernel mount point.

Reply:
> Yes. The VFS module in this case is just a helper to provide more features or improve efficiency, but most of the I/O still goes through the kernel mount point.

Ah.. more or less like _vfs_glusterfs_fuse_ module.